dev-lang/gnucobol/gnucobol-3.0_rc1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV=$(ver_cut 1-2)
+MY_P="${PN}-${MY_PV}-rc1"
```

Comment:
`MY_P=${P/_/-}` is how we usually deal with it. Especially, don't hardcode `rc1`.